eve/main.yml

```diff
@@ -98,6 +98,9 @@ stages:
         name: 'Build doc'
         haltOnFailure: true
         command: tox --workdir /tmp/tox -e docs -- html latexpdf
+        env:
+          # Fake we're building in a ReadTheDocs environment
+          READTHEDOCS: True
```

Comment:
I think that's why we're getting a weird error in eve @NicolasT

Would you mind trying the following:
```yaml
READTHEDOCS: "True"
```

Reply:
Good catch...

Reply:
:)